src/blockchain/rpc.rs

```diff
@@ -66,6 +68,8 @@ pub struct RpcBlockchain {
     capabilities: HashSet<Capability>,
     /// Sync parameters.
     sync_params: RpcSyncParams,
+    /// Import parameters.
+    import_params: RefCell<RpcImportParams>,
```

Comment:
Just to be safe I would replace this with an `Arc<RpcImportParams>`. Then you can use `Arc::make_mut()` whenever you want to replace it

Reply:
This won't work as the `WalletSync` is NOT `&mut self`. We can do `Arc<Mutex<RpcImportParams>>`? But I think this is too much. I feel like the user can just handle thread-safety elsewhere?